src/CardService.js

```diff
@@ -95,14 +98,17 @@ RiptideLab.CardService = (function(){
       return {
         add(cardName, card) {
           const exactName = card.name.toLowerCase();
-          if (cardName !== exactName) // Fuzzy matched or double faced
+          if (cardName !== exactName & !cardName.includes("|")) { // Fuzzy matched or double faced, but not exact set matches
```

Comment:
```suggestion
          if (cardName !== exactName && !cardName.includes("|")) { // Fuzzy matched or double faced, but not exact set matches
```

This should most likely be `&&` since we want the logical and not the bit-wise AND-operator

Reply:
Fixed, thank you!